tls/s2n_server_cert.c

```diff
@@ -86,6 +86,9 @@ int s2n_server_cert_recv(struct s2n_connection *conn)
 
 int s2n_server_cert_send(struct s2n_connection *conn)
 {
+    if (conn->actual_protocol_version > S2N_TLS12) {
+        GUARD(s2n_stuffer_read_uint8(&conn->handshake.io, 0));
```

Comment:
are you intending to read or write from the stuffer?

Reply:
my bad.
updated